### DISCUSSION TITLE
Having issue when date is something like 2022-12-11?

### DISCUSSION THREAD

**Discussion by EstopaceMA** [daattali/beautiful-jekyll#1070] [Q&A] [answered]
if my date is like this, it doesn't show
![image](https://user-images.githubusercontent.com/37571727/206869666-45087ab2-bdac-4db7-9880-27d838fdfb92.png)

unlike if the date is like this 2022-12-09 it shows

**Comment by daattali** [comment, marked as answer]
A post with a date in the future will probably only be shown in the future. You can't release something before it exists

**Reply by EstopaceMA** [reply]
2022-12-11 or December 11, 2022 is the current date here

**Reply by daattali** [reply]
Probably depends on the timezone for your website. Try re publishing again in a few hours.

**Reply by EstopaceMA** [reply]
Is there a configuration for the timezone?

**Reply by daattali** [reply]
https://jekyllrb.com/docs/configuration/options/

**Reply by EstopaceMA** [reply]
Thanks @daattali, I've already fixed it :octocat:
